docs: add basic docs for sub-classing awkward mixins (#468)
scikit-hep/vector: README.md

compute_masses(array)
```

## Sub-classing Awkward mixins

> At the moment, it is possible to sub-class vector awkward mixins to extend the vector functionalities. Although the mechanism is in place, it is not very user-friendly and is being worked on.

Author: ```suggestion
At the moment, it is possible to sub-class vector awkward mixins to extend the vector functionalities. Although the mechanism in place works well, it is still being improved.
```
